github/Workflow.py

```diff
@@ -106,6 +106,31 @@ def badge_url(self):
         self._completeIfNotSet(self._badge_url)
         return self._badge_url.value
 
+    def create_dispatch(self, ref, inputs):
```

Comment:
If inputs is not required, default it to NotSet.

Reply:
Done.